lib/endpoint/tags/table.js

```diff
@@ -130,6 +130,10 @@ function rowToParameter (i, el) {
     params.type = 'integer[]'
   }
 
+  if (/^dismissal_restrictions/.test(params.name)) {
```

Comment:
@knrt10 For some reason, it's not getting into this condition. Any idea why?

Reply:
@aryannarora you need to do this ```params.name = `${params.name}` ```

Reply:
@aryannarora the problem you are facing because you have to fetch latest master first and rebase it. As some code has been changed there. Then You need to change
 ``` if (/^dismissal_restrictions/.test(params.name)) {```
code according to that.

Reply:
Yeah, it appears Travis test is failing as your branch is far behind master of `octokit/routes`, try syncing your master branch with master of `octokit/routes`. When your get your master updated, fix the code again and update your PR.

Reply:
@knrt10 I guess that is not the issue here. I'm not using any code that's modified in recent commits. Still letme first resolve all the conflicts.

Reply:
@gr2m @knrt10 
I logged out all the `params.name` in a file and turns out it doesn't contains any `dismissal_restrictions.*` . Am i missing something?
Would look into it more after some sleep.